pytket/binders/include/add_gate.hpp

```diff
@@ -67,10 +67,23 @@ static Circuit *add_gate_method(
     op_signature_t sig = op->get_signature();
 
     for (unsigned i = 0; i < args.size(); ++i) {
-      if (sig.at(i) == EdgeType::Quantum) {
-        new_args.push_back(Qubit(args[i]));
-      } else {
-        new_args.push_back(Bit(args[i]));
+      switch (sig.at(i)) {
+        case EdgeType::Quantum: {
+          new_args.push_back(Qubit(args[i]));
+          break;
+        }
+        case EdgeType::WASM: {
+          new_args.push_back(WasmWireUID(args[i]));
+          break;
+        }
+        case EdgeType::Classical:
+        case EdgeType::Boolean: {
```

Comment:
Why would boolean wire not be allowed in signature of a op that is made conditional?

(From my understanding if this is removed an conditional wasm op with bool wires would fail)